### DISCUSSION TITLE
Show the duration of the category in the chat title

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5381] [Ideas]
**Suggestion**
forsen (live) — League of Legends
↓
forsen (live) — [2h 50m] League of Legends

**What for?**
There are at least 2 cases where this is useful.
1. Many streamers host events where they play different games. They allocate limited time for each game. If you are not interested in some of the games, then it will be enough to keep Chatterino open so as not to miss the moment when such games end.
2. If a streamer decides to play a game that you are not interested in watching, and you know how long it will take to complete, then you will often be able to estimate when to return to the stream (channel).

I'm not sure if this is possible right now, but I'll leave it as an idea because it has practical benefits.

**Comment by pajlada** [comment]
I'm fine with this as an option to the split naming, as long as it's clear it's a guess from our side

**Reply by Felanbird** [reply]
If we're fine with 1/4 of the `chat title` options being a drop-down, then it would be fine to add this to the existing uptime option, just changing it to a drop-down menu with `off`, `uptime`, `game(?)` and just specify in the tooltip that game* is made up data.
